README.md

```diff
@@ -195,13 +195,13 @@ docker volume create ankerctl_vol
 docker run \
   -v ankerctl_vol:/root/.config/ankerctl \
   -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-  ankerctl config import /tmp/login.json
+  ankerctl/ankerctl config import /tmp/login.json
 
 # Now that there is a /root/.config/ankerctl/default.json file that persists in the docker volume
 # we can run ankerctl without having to specify the login.json file
 docker run \
   -v ankerctl_vol:/root/.config/ankerctl \
-  ankerctl config show
+  ankerctl/ankerctl config show
```

Comment:
I'm not sure I understand this.

Is `ankerctl/ankerctl` the name of the container? Or is this a syntax I'm not familiar with?

Could we refer to `ghcr.io/...` directly here, if that's simpler?